### DISCUSSION TITLE
How can i create a workflow?

### DISCUSSION THREAD

**Discussion by gorkem1947** [community/community#134421] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

How can i create a workflow?

**Comment by all6h** [comment]
create a github/workflows Directory, create a workflow file workflow.yml

**Comment by ettaboyle** [comment]
@feraxrd @rexsql This post is considered "badge hunting" and violates our community code of conduct. You have created or participated in at least 2 posts in our community like this. This post will be locked, and your other posts are being removed and you will be blocked from participating in this community for the next 7 days. Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding. 

> We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).
> 
> **Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post.** 
>
